### ISSUE TITLE
Pylance does not suggest keys in nested TypedDicts

### ISSUE THREAD

**Issue by NCPlayz** [microsoft/pylance-release#1485]
## Environment data

-   Language Server version: v2021.6.2
-   OS and version: Microsoft Windows [Version 10.0.19043.1052]
-   Python version (& distribution if applicable, e.g. Anaconda): Python 3.9.4

## Expected behaviour

It is able to give suggestions for a nested TypedDict.

## Actual behaviour

It is not able to give suggestions, but still raises errors about not finding the item in the TypedDict.

## Logs

```
[nothing]
```

## Code Snippet / Additional information

![gif](https://i.imgur.com/y2TdBs9.gif)
![image](https://i.imgur.com/cRecQzI.png)
![image](https://i.imgur.com/Gh5jkgW.png)

```python
from typing import TypedDict

class NestedDict(TypedDict):
    value: str

class MyDict(TypedDict):
    nested: NestedDict

x: MyDict = { "nested": { "value": "Hello World!" } }
nested = x["nested"]

x["nested"]["value"]
nested["value"]

```

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next release.

**Comment by bschnurr** [comment]
This issue has been fixed in version 2021.6.3, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202163-23-june-2021
